spec/support/autoload/page_objects/assessor_interface/review_verifications.rb

```diff
@@ -11,6 +11,8 @@ class ReviewVerifications < SitePrism::Page
         element :tag, ".govuk-tag"
       end
 
+      section :task_list, TaskList, ".app-task-list"
```

Comment:
Nice, we should be able to remove lines 9-12 now as they shouldn't be being used anywhere.